kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/TaskFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.patterns.factory.tasks;
+
+public class TaskFactory {
+
+    public static final String DRIVING = "DRIVING";
+    public static final String PAINTING = "PAINTING";
+    public static final String SHOPPING = "SHOPPING";
+
+    public final Task makeTask(final String taskClass) {
```

Comment:
Zauwaz, ze ta metoda nie jest uniwersalna. A jak chce pojechac rowerem do Krakowa? Jak mi wytworzy takie zadanie?

Reply:
Jakaś wskazówka jak to zmodyfikować?